mne/coreg.py

```diff
@@ -789,6 +823,22 @@ def _write_mri_config(fname, subject_from, subject_to, scale):
 
 
 def _scale_params(subject_to, subject_from, scale, subjects_dir):
+    """Assemble parameters for scaling
```

Comment:
Some functions do, some other don't. The convention doesn't seem explicit, but the pep examples have dots:

https://www.python.org/dev/peps/pep-0257/